Makefile

```diff
@@ -23,6 +23,8 @@ test: test_build
 test-watch: compose_build
 	docker-compose -f docker-compose-test.yml run app sh -c "/app/bin/wait-for-it.sh db:5432 -- ptw -- --testmon --show-capture=no --disable-warnings"
 
+test-react: 
```

Comment:
nit: i think this should be `test_js` or at the very least `test_react`. I'm not sure why `test-watch` isn't also an underscore but it seems to be underscores everywhere else